Resources/skeleton/dto.php.twig

```diff
@@ -0,0 +1,89 @@
+<?php
+/**
+ * DTO {{ element.ucfirstName }}
+ */
+namespace {{ namespace }};
+
+use JMS\Serializer\Annotation;
+use Doctrine\Common\Collections\ArrayCollection;
+
+/**
+ * {{ element.ucfirstName }}
+ * 
+ * @Annotation\ExclusionPolicy("none")
+ * @Annotation\XmlRoot("{{ element.name }}")
+ */
+class {{ element.ucfirstName }}
+{
+    {% for node in element.children %}
+/**
+     * @var {{ node.dataType }}
+     * @Annotation\Type("{{ node.dataType }}")
+     * @Annotation\SerializedName("{{ node.name }}")
+     * @Annotation\XmlElement(cdata=false)
+     */
+    private ${{ node.name }};
+    
+    {% endfor %}
+    {% if element.firstElement == 1 %}
+/**
+     * @var string 
+     * @Annotation\Exclude()
+     */
+    private $token;
+    {% endif %}
+
+    {% for node in element.children %}
+/**
+     * Set {{ node.name }}
+     *
+     * @param {{ node.dataType }} ${{ node.name }}
+     *
+     * @return {{ element.ucfirstName }}
+     */
+    public function set{{ node.ucfirstName }}(${{ node.name }})
+    {
+        $this->{{ node.name }} = ${{ node.name }};
+
+        return $this;
+    }
+
+    /**
+     * Get {{ node.name }}
+     *
+     * @return {{ node.dataType }}
+     */
+    public function get{{ node.ucfirstName }}()
```

Comment:
As an improvement suggestion it is a good idea to detect the type of the node and if boolean to have it as isser or a hasser rather than getter.